.github/workflows/cypress.yml

```diff
@@ -0,0 +1,30 @@
+name: Cypress Tests
+
+on: [push, pull_request]
+
+jobs:
+  cypress-run:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        browsers: [chrome, firefox]
+        node-version: [14.x]
```

Comment:
Can we pin this in the package.json via the `engines` property?

```
  "engines": {
    "node": "^14.0.0",
    "npm": "^6.14.12"
  }
```
for example.

I went ahead and set  `NODE_VERSION=14` in Netlify, so the next build should use it.